mitiq/pec/tests/test_pec.py

```diff
@@ -493,6 +493,18 @@ def test_mitigate_executor_qiskit():
 
     assert np.isclose(unmitigated, mitigated)
 
+    batched_unmitigated = batched_executor([circuit])
```

Comment:
Again, is one circuit enough to fully test this?

Reply:
Good point, I remember thinking it was a bit odd when I first reviewed and then forgot to change it.